common/utils.go

```diff
@@ -89,8 +90,16 @@ func IsNil(i interface{}) bool {
 }
 
 func LogOnPanic() {
-	if err := recover(); err != nil {
-		logutils.ZapLogger().Error("panic in goroutine", zap.Any("error", err), zap.Stack("stacktrace"))
-		panic(err)
+	err := recover()
+	if err == nil {
+		return
 	}
+
+	logutils.ZapLogger().Error("panic in goroutine",
+		zap.Any("error", err),
+		zap.Stack("stacktrace"))
+
+	sentry.RecoverError(err)
```

Comment:
Similar to another comment @igor-sirotin shouldn't this call be protected by a condition in case the user hasn't allowed Sentry to do its thing?